dashboard/test/ui/config/courses/ui-test-course-2017.course

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ui-test-course-2017",
+  "script_names": [
+    "ui-test-script-2017"
```

Comment:
I love the `ui-test-` prefix! I had a hard time figuring out what each script was for in this PR, though. would you be willing to name ui-test-script-2017 to something like ui-test-script-in-course-2017, ui-test-course-unit-2017, or even ui-test-course-script-2017 (and same for -2019)? Otherwise, the current name kind of sounds a bit too much like the opposite, i.e. a script not in a course.